src/Xamarin.Android.Build.Tasks/Tasks/Aot.cs

```diff
@@ -186,6 +183,10 @@ IEnumerable<Config> GetAotConfigs (NdkTools ndk)
 							break;
 					}
 
+					if (!string.IsNullOrEmpty (LdFlags)) {
+						aotOptions.Add ($"ld-flags={LdFlags}");
+					}
+
```

Comment:
Do we also want to move the ld-name arg down here? I see there is a comment above about making sure both are always passed last.

Reply:
Not sure, to be honest. `ld-name` is an argument which takes a simple value which doesn't appear to confuse the AOT parser. I think I'd leave it where it is, especially that everything seems to have worked fine after the `ld-flags` repositioning.